packages/cf.js/src/types/events.ts

```diff
@@ -33,6 +33,7 @@ export interface UpdateStateEventData extends AppEventData {
 export interface ErrorEventData {
   errorName: string;
   message?: string;
+  extra?: { [k: string]: any };
```

Comment:
@mvanderh I think we can restrict a bit the type here without being too overly restraining:

```ts
extra?: { [k: string]: string | number | boolean | object | null }
```

Reply:
I like it. 

Btw you can suggest code changes that i can immediately accept or decline with Cmd-G.

Reply:
Yeah, I keep forgetting that feature. Thanks for the reminder :D